.travis.yml

```diff
@@ -27,4 +27,4 @@ after_success:
 - bash <(curl -s https://codecov.io/bash)
 
 go:
-- 1.8.3
+- 1.9
```

Comment:
For the time being we can enable both @balamurugana ? i.e keep both `1.8.3` and `1.9`